### ISSUE TITLE
Navigation menu: avoid all-caps text

### ISSUE THREAD

**Issue by afercia** [WordPress/twentytwentyone#469]
**Describe the bug**
All-cap text is known to be harder to read, especially for some users, for example users with dyslexia, and it's generally less readable for all users. Worth reminding dyslexia comes in many different types and what may be good for some users may not be good for others. Ideally, there should be a mechanism to customize many typeface metrics including spacing between characters, spacing between words, line-height, and the ability to disable all-caps. Given that this is not possible, the best option is to provide safe defaults and avoid all-caps.

Also, all-caps text causes some screen readers to spell the words instead of speaking them as words.

As far as I can tell, the Web Content Accessibility Guidelines don't explicitly mention all-caps text. However, it is mentioned in the W3C recommendation for Low Vision:

Accessibility Requirements for People with Low Vision
3.3.4 Capitalization
https://www.w3.org/TR/low-vision-needs/#capitalization
> Text in all capital letters is more difficult to read for most people, with and without disabilities.

This is still a W3C "Working Draft" so, technically, it's not normative. That's what we have.

Other authoritative references:

Harvard University
https://accessibility.huit.harvard.edu/design-readability
> Avoid using all caps. Readability is reduced with all caps because all words have a uniform rectangular shape, meaning readers can't identify words by their shape.

WebAIM (Center for Persons with Disabilities Utah State University)
https://webaim.org/articles/evaluatingcognitive/
> Do not use italics or bold on long sections of text. Be careful with ALL CAPS.

Some accessibility auditing tools do even have a specific rule to detect all-caps text, for example the tool from the UK Government Digital Service:
https://alphagov.github.io/accessibility-tool-audit/tests/typography-all-caps-text-found.html

I do realize removing the all-caps text has an impact on the design. However, if we want to provide users with the highest possible level of accessibility I'd strongly recommend to reconsider the design choice to use all-caps and use normal text instead. 

**To Reproduce**
- see the all-caps text in the navigation menu

**Comment by aristath** [comment]
PR on #474 
Pinging @kjellr and @melchoyce since this will require design feedback :+1:

**Comment by afercia** [comment]
One more reference, specific to dyslexia:

British Dyslexia Association
https://www.bdadyslexia.org.uk/advice/employers/creating-a-dyslexia-friendly-workplace/dyslexia-friendly-style-guide
> Avoid text in uppercase/capital letters and small caps, which can be less familiar to the reader and harder to read.

**Comment by kjellr** [comment]
This seems ok on my end. Changing this doesn't drastically impact the overall impression of the theme: 

Before|After
---|---
![iPad mini - 6](https://user-images.githubusercontent.com/1202812/96010760-2b151680-0e10-11eb-9151-ed6218367044.png)|![iPad mini - 1](https://user-images.githubusercontent.com/1202812/96010466-cf4a8d80-0e0f-11eb-9fb1-05f6bdfcc32f.png)|

The only design challenge this would introduce is how to handle menu descriptions. As of #424, the visual differentiator between menu items and their descriptions is just the text case: 

![iPad mini - 5](https://user-images.githubusercontent.com/1202812/96009889-2bf97880-0e0f-11eb-8903-f887ff2a496b.png)

If we change the menu items so that they're not all caps, I'd suggest that we make the description text slightly smaller instead. ~14px works fine: 

![iPad mini - 3](https://user-images.githubusercontent.com/1202812/96010166-81358a00-0e0f-11eb-8cd6-54124f646b10.png)

---

In any case, I know @melchoyce is unavailable at the moment. I don't think she'll have any major objections either, but if we can hold off until tomorrow to merge so she can take a look, that would be great.

**Comment by melchoyce** [comment]
I think it would work better if we increased the font size of the navigation items to 20px, rather than decreasing the size of the description:

![image](https://user-images.githubusercontent.com/2846578/96132406-6115d180-0ec8-11eb-9623-c6d7303ee740.png)

**Comment by aristath** [comment]
Perfect, thank you @melchoyce 
I'll go ahead and merge #474, and create a separate ticket to increase the font-size for menus.

**Comment by aristath** [comment]
Issue for the font-size: https://github.com/WordPress/twentytwentyone/issues/505
